### PR TITLE
feat(syslogexporter): remove adding additional structure data in syslog exporter

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
Token which is required by Cloud Syslog Source can be added by logtransform procesor

### CODE REVIEW

File: pkg/exporter/syslogexporter/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 The syslog exporter supports sending messages to a remote syslog server.
 
 - This exporter can forward syslog messages to a third party [syslog server](https://www.rsyslog.com/) using RFC5424 and RFC3164
-- It also supports sending syslog messages to the [Cloud Syslog Source](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-syslog-source/) configured on a Sumo Logic hosted collector using the RFC5424 format
+- It also supports sending syslog messages to the [Cloud Syslog Source](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-syslog-source/) configured on a Sumo Logic hosted collector using the RFC5424 format, token required by Cloud Syslog Source can be added using logstransform processor, please see [example configuration](./examples/config_with_token.yaml)
```
Comment (sumo-drosiek) [inline pkg/exporter/syslogexporter/README.md:10]: could we link `[logstransform processor]`?

**Review by swiatekm-sumo — APPROVED** [review]

File: pkg/exporter/syslogexporter/README.md (hunk 1 above)
Comment (sumo-drosiek) [reply pkg/exporter/syslogexporter/README.md:10]: also RFC5424 -> `RFC5424`

File: pkg/exporter/syslogexporter/examples/config_with_token.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    - /other/path/**/*.txt
+    operators:
+      - type: syslog_parser
+        protocol: rfc5424
+processors:
+  logstransform:
+    operators:
+      - type: add
+        field: attributes.message
+        value: EXPR("YOUR_TOKEN " + attributes.message)
```
Comment (sumo-drosiek) [inline pkg/exporter/syslogexporter/examples/config_with_token.yaml:58]: nice!

File: pkg/exporter/syslogexporter/syslog_test.go [hunk 3]
```diff
@@ -72,28 +56,3 @@ func TestFormatRFC5424(t *testing.T) {
 	assert.Equal(t, true, strings.Contains(formattedMsg, "UserID=\"Tester2\""))
 	assert.Equal(t, true, strings.Contains(formattedMsg, "PEN=\"27389\""))
 }
-
-func TestFormatRFC5424NoAddStructuredData(t *testing.T) {
```
Comment (rnishtala-sumo) [inline pkg/exporter/syslogexporter/syslog_test.go:76]: should we keep this test?

Comment (kkujawa-sumo) [reply pkg/exporter/syslogexporter/syslog_test.go:76]: I keep this test  but I merged tests which were in two functions. I left test cases which are relevant after deleting support for adding additional structured data.

File: pkg/exporter/syslogexporter/README.md (hunk 1 above)
Comment (kkujawa-sumo) [reply pkg/exporter/syslogexporter/README.md]: done :)  @sumo-drosiek please take a look once again

**Review by rnishtala-sumo — APPROVED** [review]

**Pull request merged**